### ISSUE TITLE
list context and row wise compute through the javascript API

### ISSUE THREAD

**Issue by schicks** [pola-rs/nodejs-polars#21]
I also asked this on stackoverflow [here](https://stackoverflow.com/questions/74843519/node-js-polars-list-context-and-row-wise-compute), but as primarily I'm asking for help finding a feature that definitely does exist through the python bindings I think this may be a more appropriate place.

I'm trying to use polars to calculate the results of ranked choice elections in a hypothetical space. I was able to get this working through the python bindings [here](https://gist.github.com/schicks/5ea4085acf6312e0281ec48a103fc1d5) without too much trouble, but it depends on [list context](https://pola-rs.github.io/polars-book/user-guide/dsl/list_context.html) features, specifically `pl.element()`, in order to do some of the calculations. The problem I'm running into is that there does not appear to be an equivalent feature in the nodejs bindings. Is there a different way to talk about the element of a list? Is there a way to do something like this;
```python
lambda loser: pl.col("vote").arr.eval(pl.element().filter(pl.element() != loser)).alias("vote")
```
within nodejs-polars, or without using `pl.element()`?

**Comment by universalmind303** [comment]
We can definitely add `pl.element` to the JS bindings! 

As a note, the `pl.element()` is just syntactic sugar for `pl.col("")` . So in the meantime, you should be able to use that as a workaround. 

Right now, we are partially blocked on deploying new versions of the node bindings as a lot of the CI images are broken. See https://github.com/napi-rs/napi-rs/issues/1389. I currently don't have the time/expertise to create custom images to run the CI on though.

**Comment by schicks** [comment]
That's good to know! Thanks for the stopgap advice.

**Comment by Bidek56** [comment]
This issue was resolved with #28
